wizeline/remix-project-lab
# Feat/201/create an application form and allow sign in for non wizeline email users

# What does this PR do?
Create an application form to fill out the Applicants table information



# Where should the reviewer start?
/app/routes/applicationForm/$applicantId/index.tsx
/app/routes/applicationForm/$applicantId/createapplicant.tsx



# How should this be manually tested?
1. Go to /applicationForm/"YourProfileId"
2. Fill in the form fields and submit it
3. You will be redirected to /applicants
4. Check if your information is there



# Any background context you want to provide?



# What are the relevant tickets?
#201 



# Screenshots
<img width="1207" alt="Captura de pantalla 2023-09-04 a la(s) 10 41 06 p m" src="https://github.com/wizeline/remix-project-lab/assets/58340818/1dc44f68-5a2d-4641-8d69-d8c0a8a4e7dc">
<img width="1207" alt="Captura de pantalla 2023-09-04 a la(s) 10 41 16 p m" src="https://github.com/wizeline/remix-project-lab/assets/58340818/08acbd07-a955-4328-bff0-b5877aaf8ebf">
<img width="1207" alt="Captura de pantalla 2023-09-04 a la(s) 10 41 30 p m" src="https://github.com/wizeline/remix-project-lab/assets/58340818/f1bbf6bf-1610-4bed-bc97-7f00bc05069d">



# Questions



# Checklist
